doc/rados/configuration/network-config-ref.rst

```diff
@@ -114,7 +114,7 @@ Each Ceph OSD Daemon on a Ceph Node may use up to four ports:
 
 #. One for talking to clients and monitors.
 #. One for sending data to other OSDs.
-#. Two for heartbeating on each interface.
+#. Two for heart beating on each interface.
```

Comment:
I think "heartbeating" as one word is appropriate here, i.e. apply no change.  @zdover23 ?

Reply:
Oh, I meant to grab those, not sure why they got lost. I'll drop that commit.